build/grafana/dashboard-status.yaml

```diff
@@ -1309,111 +1321,13 @@ data:
           ],
           "repeat": null,
           "repeatDirection": "h",
-          "repeatIteration": 1563127715245,
```

Comment:
@cyriltovena everything seems fine, but any idea why this gets deleted?

I _think_ it might be repeated in the previous version?